janitor/functions.py

```diff
@@ -6151,28 +6185,6 @@ def pivot_longer(
         repeated as necessary.
     :returns: A pandas DataFrame that has been unpivoted from wide to long
         format.
-    :raises TypeError: if `index` or `column_names` is not a string, or a
-        list/tuple of column names, or a `janitor.patterns` function.
```

Comment:
all the Errors are removed. They are no longer valid ?

Reply:
darglint only issues errors if they are within the docs (since there are no error handling within functions.py, I removed them). The actual error handlings are implemented in utils.py

Reply:
Ok 👍